fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordContext.java

```diff
@@ -152,22 +154,23 @@
     private final Map<String, PostCommit> postCommits = new LinkedHashMap<>();
     private boolean dirtyStoreState;
     private boolean dirtyMetaDataVersionStamp;
+    private long trackOpenTimeNanos;
 
     protected FDBRecordContext(@Nonnull FDBDatabase fdb,
                                @Nonnull Transaction transaction,
-                               @Nonnull FDBRecordContextConfig config,
-                               boolean transactionIsTraced) {
+                               @Nonnull FDBRecordContextConfig config) {
         super(fdb, transaction, config.getTimer());
         this.transactionCreateTime = System.currentTimeMillis();
         this.localVersion = new AtomicInteger(0);
         this.localVersionCache = new ConcurrentSkipListMap<>(ByteArrayUtil::compareUnsigned);
         this.versionMutationCache = new ConcurrentSkipListMap<>(ByteArrayUtil::compareUnsigned);
         this.transactionId = getSanitizedId(config);
+        this.openStackTrace = config.isSaveOpenStackTrace() ? new Throwable() : null;
```

Comment:
Seems like the throwable here should have a message here about how it's not thrown (or equivalently that this is created only for the stack trace).

Reply:
Added.